app/views/content/landing/food-for-thought/_promo.html.erb

```diff
@@ -5,7 +5,7 @@
       <% promo.right_section(
         heading: "Find out what teaching is really like"
       ) do %>
-          <p>Talk to an experienced former teacher who can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.</p>
+          <p>Talk to an experienced former teacher for free who can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.</p>
```

Comment:
```suggestion
          <p>Talk to an experienced former teacher for free. They can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.</p>
```
This sentence feels a bit clunky at the moment - does it need rewriting slightly (my suggestion can probably be improved!)/